### ISSUE TITLE
Norway tests failing

### ISSUE THREAD

**Issue by badams** [azuyalabs/yasumi#11]
Some tests for the Norway provider are failing on my local machine, I've tracked it down to the timezone being set inconsistently between `Yasumi\Provider\Norway` and `Yasumi\Tests\Norway\NorwayBaseTestCase`

Output from PHPUnit:
```
PHPUnit 4.8.24 by Sebastian Bergmann and contributors.

Runtime:	PHP 5.5.30 with Xdebug 2.3.3
Configuration:	/Users/byron/projects/badams/yasumi/phpunit.xml

.............................................................   61 / 1166 (  5%)
.............................................................  122 / 1166 ( 10%)
.............................................................  183 / 1166 ( 15%)
.............................................................  244 / 1166 ( 20%)
.............................................................  305 / 1166 ( 26%)
.............................................................  366 / 1166 ( 31%)
.............................................................  427 / 1166 ( 36%)
.............................................................  488 / 1166 ( 41%)
.............................................................  549 / 1166 ( 47%)
.............................................................  610 / 1166 ( 52%)
.............................................................  671 / 1166 ( 57%)
....F...FF...F.F.............FF..F.F.F.F.F....FF...........FF  732 / 1166 ( 62%)
.F..FF.......................................................  793 / 1166 ( 68%)
.............................................................  854 / 1166 ( 73%)
.............................................................  915 / 1166 ( 78%)
.............................................................  976 / 1166 ( 83%)
............................................................. 1037 / 1166 ( 88%)
............................................................. 1098 / 1166 ( 94%)
............................................................. 1159 / 1166 ( 99%)
.......

Time: 3.5 seconds, Memory: 41.00Mb

There were 19 failures:

1) Yasumi\Tests\Norway\AscensionDayTest::testAscensionDay
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1877-05-10T00:00:00+0043
+1877-05-10T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/AscensionDayTest.php:35

2) Yasumi\Tests\Norway\ChristmasDayTest::testHoliday with data set #2 ('1410', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1410-12-25T00:00:00+0043
+1410-12-25T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/ChristmasDayTest.php:37

3) Yasumi\Tests\Norway\ChristmasDayTest::testHoliday with data set #3 ('1641', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1641-12-25T00:00:00+0043
+1641-12-25T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/ChristmasDayTest.php:37

4) Yasumi\Tests\Norway\ChristmasDayTest::testHoliday with data set #7 ('1171', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1171-12-25T00:00:00+0043
+1171-12-25T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/ChristmasDayTest.php:37

5) Yasumi\Tests\Norway\ChristmasDayTest::testHoliday with data set #9 ('1557', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1557-12-25T00:00:00+0043
+1557-12-25T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/ChristmasDayTest.php:37

6) Yasumi\Tests\Norway\InternationalWorkersDayTest::testInternationalWorkersDay with data set #3 ('1895', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1895-05-01T00:00:00+0043
+1895-05-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/InternationalWorkersDayTest.php:37

7) Yasumi\Tests\Norway\InternationalWorkersDayTest::testInternationalWorkersDay with data set #4 ('1205', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1205-05-01T00:00:00+0043
+1205-05-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/InternationalWorkersDayTest.php:37

8) Yasumi\Tests\Norway\InternationalWorkersDayTest::testInternationalWorkersDay with data set #7 ('1150', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1150-05-01T00:00:00+0043
+1150-05-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/InternationalWorkersDayTest.php:37

9) Yasumi\Tests\Norway\InternationalWorkersDayTest::testInternationalWorkersDay with data set #9 ('1621', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1621-05-01T00:00:00+0043
+1621-05-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/InternationalWorkersDayTest.php:37

10) Yasumi\Tests\Norway\MaundyThursdayTest::testHoliday
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1655-04-12T00:00:00+0043
+1655-04-12T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/MaundyThursdayTest.php:35

11) Yasumi\Tests\Norway\NewYearsDayTest::testHoliday with data set #0 ('1358', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1358-01-01T00:00:00+0043
+1358-01-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/NewYearsDayTest.php:37

12) Yasumi\Tests\Norway\NewYearsDayTest::testHoliday with data set #2 ('1705', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1705-01-01T00:00:00+0043
+1705-01-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/NewYearsDayTest.php:37

13) Yasumi\Tests\Norway\NewYearsDayTest::testHoliday with data set #7 ('1683', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1683-01-01T00:00:00+0043
+1683-01-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/NewYearsDayTest.php:37

14) Yasumi\Tests\Norway\NewYearsDayTest::testHoliday with data set #8 ('1707', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1707-01-01T00:00:00+0043
+1707-01-01T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/NewYearsDayTest.php:37

15) Yasumi\Tests\Norway\SecondChristmasDayTest::testHoliday with data set #0 ('1138', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1138-12-26T00:00:00+0043
+1138-12-26T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/SecondChristmasDayTest.php:37

16) Yasumi\Tests\Norway\SecondChristmasDayTest::testHoliday with data set #1 ('1894', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1894-12-26T00:00:00+0043
+1894-12-26T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/SecondChristmasDayTest.php:37

17) Yasumi\Tests\Norway\SecondChristmasDayTest::testHoliday with data set #3 ('1280', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1280-12-26T00:00:00+0043
+1280-12-26T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/SecondChristmasDayTest.php:37

18) Yasumi\Tests\Norway\SecondChristmasDayTest::testHoliday with data set #6 ('1586', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1586-12-26T00:00:00+0043
+1586-12-26T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/SecondChristmasDayTest.php:37

19) Yasumi\Tests\Norway\SecondChristmasDayTest::testHoliday with data set #7 ('1767', DateTime Object (...))
Failed asserting that two DateTime objects are equal.
--- Expected
+++ Actual
@@ @@
-1767-12-26T00:00:00+0043
+1767-12-26T00:00:00+0050

/Users/byron/projects/badams/yasumi/tests/YasumiBase.php:89
/Users/byron/projects/badams/yasumi/tests/Norway/SecondChristmasDayTest.php:37

FAILURES!
Tests: 1166, Assertions: 5393, Failures: 19.
```

**Comment by stelgenhof** [comment]
Merged PR #12 solving this issue.

**Comment by stelgenhof** [comment]
Merged PR #12 solving this issue.
